src/foam/dao/journal/FileAppendDAO.js

```diff
@@ -0,0 +1,100 @@
+/**
+ * @license
+ * Copyright 2018 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.dao.journal',
+  name: 'FileAppendDAO',
+  extends: 'foam.dao.AbstractDAO',
```

Comment:
I think it's confusing to make this a DAO when it only supports two methods. Better to make a separate Journal interface.

Reply:
It's nice for this to be a DAO because we can swap it out with any other DAO for testing purposes or we can decorate it with any of the many DAO decorators we already have. It's already being decorated in the EasyJournalEntryDAO to get sequence numbers on the journal entries.